tasks/src/reinforcement-learning/about.md

```diff
@@ -1,6 +1,14 @@
 ## Use Cases
 
-You can contribute this area with common use cases of the task! 
+### Gaming
+
+Reinforcement learning is known for its application to video games. Since the games provide a safe environment to the agent to be trained in the sense that it is perfectly defined and controllable. This makes them perfect candidates for experimentation and will help a lot to learn about the capabilities and limitations of various RL algorithms. 

```

Comment:
I really liked how you put it, someone else is adding the same case but we can prepend or append your definition (if that one gets merged, or we can simply only have this.)

Reply:
@merveenoyan I am glad you liked it. I will try to add more as long as I find time for that :)